docs/fundamentals/platform.rst

```diff
@@ -1,2 +1,279 @@
+
+    platform.turn_on_instruments()
+    # Turns on the signal output for the generator instruments (RF, voltage sources and current sources).
+    # This does not actually turn on the instruments of the laboratory, it only opens the signal output generation of the sources.
+    # You might want to skip this step if the instruments outputs are already open, but again, we recommend doing it anyway.
+
+.. note::
+
+    To connect, your computer must be in the same network of the instruments specified in the runcard (with their IP's addresses).
+
+|
+
+Executing a circuit with Platform:
+-----------------------------------
+The Platform offers the capability to execute circuits defined with `Qibo <https://qibo.science/>`_, an open-source middleware for quantum computing.
+
+To execute a circuit, you first need to build, connect, and set up the platform as shown in the above examples. Then, define your
+Qibo circuit, for example, a pi pulse and a measurement gate on qubit ``q`` (``int``):
+
+.. code-block:: python3
+
+    from qibo.models import Circuit
+    from qibo import gates
+
+    circuit = Circuit(q+1)
+    circuit.add(gates.X(q))
+    circuit.add(gates.M(q))
+
+And you are ready to execute the circuit with the platform:
+
+>>> result = platform.execute(program=circuit, num_avg=1000, repetition_duration=6000)
+>>> result.array
+array([[5.],
+        [5.]])
+TODO: !!! Change this results for the actual ones !!!
+
+getting the integrated values of the I/Q signals received by the digitizer!
+
+.. note::
+
+    When disabling scope acquisition mode, the array obtained has shape `(#sequencers, 2, #bins)`. In this case,
+    given that you are using only 1 sequencer to acquire the results, you would obtain an array with shape `(2, #bins)`.
+
+|
+
+Running a Rabi sequence with Platform:
+---------------------------------------
+
+To perform a Rabi sequence, build, connect and set up the platform, and then create a circuit with a
+pi pulse and a measurement gate in qubit ``q`` (``int``), as in the previous examples, which all together look like:
+
+.. code-block:: python
+
+    import qililab as ql
+
+    import numpy as np
+
+    from qibo.models import Circuit
+    from qibo import gates
+
+    # Defining the Rabi circuit:
+    circuit = Circuit(q+1)
+    circuit.add(gates.X(q))
+    circuit.add(gates.M(q))
+
+    # Building the platform:
+    platform = ql.build_platform(runcard="runcards/galadriel.yml")
+
+    # Connecting and setting up the platform:
+    platform.connect()
+    platform.initial_setup()
+    platform.turn_on_instruments()
+
+Now to run the Rabi sequence, you would need to run this sequence by looping over the gain of the AWG used
+to create the pi pulse:
+
+.. image:: platform_images/rabi.png
+  :width: 400
+  :align: center
+
+To do this, you need to use the ``set_parameter()`` method with the alias of the bus used
+to drive qubit ``q`` (let's assume it's called ``"drive_q"``):
```

Comment:
afaik we run the rabi by changing the max amplitude of the Drag gate - though it's true that changing the Gain would also work
checkout the code from `LabScripts/Personal Folders/Qililab/rabi.ipynb`
```
pulses = []

# Run experiment
results_list = []
for amp in tqdm(iterable=AMP_VALUES, total=len(AMP_VALUES), desc="Amplitude"):
    platform.set_parameter(alias=f"Drag({QUBIT})", parameter=ql.Parameter.AMPLITUDE, value=float(amp))
    result = platform.execute(program=circuit, num_avg=HW_AVG, repetition_duration=REPETITION_DURATION)
    results_list.append(result.array)

results = np.hstack(results_list)
```

Reply:
I'll change it then!